keylime/tenant.py

```diff
@@ -1010,6 +1041,34 @@ def main(argv=sys.argv):
     if args.verifier_ip is not None:
         mytenant.verifier_ip = args.verifier_ip
 
+    # we only need to fetch remote files if we are adding or updateing
+    if args.command in ['add', 'update']:
+        delete_tmp_files = logger.level > logging.DEBUG # delete tmp files unless in DEBUG mode
+
+        if args.allowlist_url:
+            logger.info("Downloading Allowlist from %s" % (args.allowlist_url))
```

Comment:
nit: Consider using `logging`s string templating directly here and in other log statements.

```suggestion
            logger.info("Downloading Allowlist from %s", args.allowlist_url)
```

Reply:
Agreed and fixed. This is something we also need for the rest of keylime, but I'll do that in another PR